cli/src/generate/grammar-schema.json

```diff
@@ -57,6 +60,15 @@
     "word": {
       "type": "string",
       "pattern": "^[a-zA-Z_]\\w*"
+    },
+
+    "supertypes": {
+      "description": "each supertype instructs the parser to replace a set of subtypes in the `types` list with a single supertype.",
+      "type": "array",
+      "items": {
+        "description": "the name of a rule in `rules` or `extras`",
```

Comment:
I would just remove this description. I don't think it makes sense to put substantial documentation directly into this grammar schema file, since most users won't look at this. I think that if anything, there should be a few links to the main documentation site. I suggested putting one link in the description of the parent object.

Reply:
I practically agreed; I've merged your changes.
I feel that the schema should be the source of truth about the JSON grammar structure. However, I don't feel strongly enough to automate making legible docs out of the schema.  
Thanks for the PR review!